docker/Dockerfile

```diff
@@ -12,17 +12,11 @@ RUN ln -s /opt/conda/etc/profile.d/conda.sh /etc/profile.d/conda.sh && \
 RUN apt-get update
 RUN apt-get install -y git git-lfs jq
 
-RUN git clone https://github.com/pytorch/functorch /workspace/functorch
```

Comment:
@xuzhao9 is the docker image still used? Do you have a suggestion of where/how to test that this is working correctly?

also, are there any CI tests for dynamo?

Reply:
Currently this docker image is not used, it is only for external enthusiasts who are interested in testing TorchBench in docker environment.

I believe @desertfire is working on CI tests for dynamo.